src/window.js

```diff
@@ -46,7 +69,17 @@ var ToolboxWindow = GObject.registerClass({
         });
         this._toolbox_frame.get_style_context().add_class('toolbox');
 
-        Gtk.Container.prototype.add.call(this, this._lockscreen);
+        let container = this._lockscreen;
+        if (GLib.getenv('HACK_TOOLBOX_HACK_MODE_ENABLED') &&
+                this.target_app_id == 'com.endlessm.HackUnlock') {
```

Comment:
Let's fix the eslint problems before merge, you should use `===` instead of `==`.